.env.docker

```
@@ -552,21 +553,20 @@ MCP_WS_TLS=false
 MCP_AUTH_JWT_ALGORITHMS=RS256
 MCP_AUTH_JWT_AUDIENCE=http://localhost:3001
-MCP_AUTH_JWT_ISSUER=gauzy-dev-auth
+MCP_AUTH_JWT_ISSUER=http://localhost:3003
 
 # Option 1: Use JWKS URI for dynamic key discovery (recommended)
-MCP_AUTH_JWT_JWKS_URI=http://localhost:3001/.well-known/jwks.json
+MCP_AUTH_JWT_JWKS_URI=http://localhost:3003/.well-known/jwks.json
 
 # Docker development authorization server configuration
-MCP_AUTH_SERVERS=[{"issuer":"gauzy-dev-auth","authorizationEndpoint":"http://localhost:3001/oauth2/authorize","tokenEndpoint":"http://localhost:3001/oauth2/token","grantTypesSupported":["authorization_code","client_credentials","refresh_token"],"responseTypesSupported":["code"],"scopesSupported":["mcp.read","mcp.write","mcp.admin"],"codeChallengeMethodsSupported":["S256"]}]
+MCP_AUTH_SERVERS=[{"issuer":"http://localhost:3003","authorizationEndpoint":"http://localhost:3001/oauth2/authorize","tokenEndpoint":"http://localhost:3001/oauth2/token","grantTypesSupported":["authorization_code","client_credentials","refresh_token"],"responseTypesSupported":["code"],"scopesSupported":["mcp.read","mcp.write","mcp.admin"],"codeChallengeMethodsSupported":["S256"]}]
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)